cmake/zycore-config.cmake.in

```diff
@@ -2,6 +2,13 @@ set(zycore_VERSION @PROJECT_VERSION@)
 
 @PACKAGE_INIT@
 
+include(CMakeFindDependencyMacro)
+if (${CMAKE_SYSTEM_NAME} STREQUAL "Linux" AND NOT @ZYAN_NO_LIBC@)
+    find_dependency(Threads)
```

Comment:
Why do you need the condition?

Reply:
I just mirrored the condition from [CMakeLists.txt](https://github.com/zyantific/zycore-c/blob/master/CMakeLists.txt#L190-L194). `Threads` is not always required as dependency.

Reply:
Alrighty!